pkg/portal/portal.go

```diff
@@ -160,22 +161,36 @@ func (p *portal) setupRouter(kconfig *kubeconfig.Kubeconfig, prom *prometheus.Pr
 	allGroups := append([]string{}, p.groupIDs...)
 	allGroups = append(allGroups, p.elevatedGroupIDs...)
 
-	p.aad, err = middleware.NewAAD(p.log, p.audit, p.env, p.baseAccessLog, p.hostname, p.sessionKey, p.clientID, p.clientKey, p.clientCerts, allGroups, unauthenticatedRouter, p.verifier)
+	// we get the env var from within the function because it is not meant to be configurable
+	disableOauthOption := os.Getenv("DISABLE_OAUTH")
+	if disableOauthOption == "true" || env.IsCI() || env.IsLocalDevelopmentMode() {
```

Comment:
Since we will only ever test this in INT (since we don't/can't run the portal e2e in PROD), can we make it so that we need to give the flag AND be in local development mode/CI for this to work? So, I guess, `disableOauthOption == "true" && (env.IsCI() || env.IsLocalDevelopmentMode())`?

Reply:
pr e2e is actually running with local development mode, I had that condition before but it wasn't working. Something like `(disableOauthOption == "true" && env.IsCI()) || env.IsLocalDevelopmentMode()` should work though